scripts/release-notes/index.js

```diff
@@ -317,10 +372,26 @@ ${formatCommits(commits)}
 `);
 };
 
+const extractCoAuthorsFromMessage = (message) => {
```

Comment:
As I [mentioned on the issue thread](https://github.com/medic/cht-core/issues/10023#issuecomment-2968207202) last month, I think it would be better to just use the GitHub GraphQL api to get the commit authors instead of parsing the commit message with regex. 

Practically speaking, I think we need to update the query above at the current line `119`.  We should replace `author { user { login, name, url } }` with:

```
        authors(first: 50) {
          edges {
            node { user { login, name, url } }
          }
        }
```

Then we just need to update the down-stream code to use the `authors` array from the response instead of the `author` field.